cli/src/commands/bookmark/create.rs

```diff
@@ -53,7 +53,9 @@ pub fn cmd_bookmark_create(
     let mut workspace_command = command.workspace_helper(ui)?;
     let target_commit = workspace_command
         .resolve_single_rev(ui, args.revision.as_ref().unwrap_or(&RevisionArg::AT))?;
+    let repo = workspace_command.repo();
     let view = workspace_command.repo().view();
+    let is_hidden = target_commit.is_hidden(repo.as_ref())?;
```

Comment:
nit: I think `is_hidden()` test is redundant (in all commands.) iirc, these commands check if bookmarks are moved/created, so we can just do `.add_head()` if there are any bookmark changes.

Reply:
True, the repo seems like a better level to do it. I've updated my branch. I should also add tests for it in jj-lib. Or do you want to take my commit and add tests, @PhilipMetzger?

Reply:
> I think `is_hidden()` test is redundant (in all commands.) iirc, these commands check if bookmarks are moved/created, so we can just do `.add_head()` if there are any bookmark changes.

I agree, but it was the simplest way to implement it. 

> True, the repo seems like a better level to do it. I've updated my branch. I should also add tests for it in jj-lib. Or do you want to take my commit and add tests, @PhilipMetzger?

I in general agree with the direction we take here, so I probably should take over Martin's branch and drop the first commit, if it otherwise doesn't provide value.